pkg/asset/internal/templates.go

```diff
@@ -404,6 +404,7 @@ spec:
         command:
         - ./hyperkube
         - controller-manager
+        - --use-service-account-credentials
```

Comment:
fyi @colemickens this is a requirement for using pod security policies.

Reply:
Sage []()